app/build.gradle

```diff
@@ -35,6 +35,8 @@ android {
         buildConfigField "String", "GIT_SHA", "\"${gitSha}\""
         buildConfigField "String", "BUILD_TIME", "\"${buildTime}\""
 
+        resConfigs "en", "cs", "de", "es", "fr", "hi", "ko", "lt", "nl", "pl", "pt-rBR", "ru", "sk", "th", "tr", "uk", "zh-rCN", "zh-rHK", "zh-rTW"
```

Comment:
This looks like a potential problem in maintenance as the values-xx folders and this line could diverge. Hope they will figure out a way to have an "auto" in place. From what I see in the bug tracker this is not there yet.